Lib/test/support/__init__.py

```diff
@@ -22,8 +21,34 @@
 import unittest
 import warnings
 
+from .os_helper import FS_NONASCII
```

Comment:
Can you please group these imports? from .os_helper import (FS_NONASCII, SAVEDCWD, ...)?

Reply:
updated.